apps/docs/src/common/pages/get-started/start-prototype.tsx

```diff
@@ -0,0 +1,103 @@
+import { FC, Fragment, createElement as h } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { PageProps } from '@not-govuk/app-composer';
+import { A } from '@not-govuk/components';
+const applyPrototype1 = require('../../../../assets/images/prototype-screen01.png').default;
+const applyPrototype2 = require('../../../../assets/images/prototype-screen02.png').default;
+
+
+const Page: FC<PageProps> = ({ location }) => (
+  <div className="govuk-grid-row">
+    <Helmet>
+      <title>Guidance on Home Office Design System</title>
+      <meta name="og:article:section" content="Get started" />
+    </Helmet>
+    <div className="govuk-grid-column-one-third">
+      <aside>
+      <h2>Prototyping</h2>
+        <ul className="plain">
+          <li><A href="/get-started/start-prototype">Starting a new prototype</A></li>
+          <li><A href="/get-started/use-prototype">Using the components</A></li>
+          <li><A href="/get-started/deploy-prototype">Deploying</A></li>
+        </ul>
+      </aside>
+    </div>
+
+    <div className="govuk-grid-column-two-thirds">
+      <h1>
+        <span className="caption">Prototyping</span>
+        Starting a new prototype
+      </h1>
+      <p>Rapidly create HTML prototypes of internal Home Office services. </p>
+
+      <div className="govuk-inset-text">
+      If your team is planning to use the HODS production framework, you
+      might consider having the developers create a blank 'app' within the
+      project to make it easier to transform your prototype into a production
+      application.
+      </div>
+
+      <h2>Before you start</h2>
+        <p>You'll need:</p>
+        <p><A href="https://nodejs.org/en/">Node.js</A> - version 12.x.x or higher</p>
+
+        <p>For the purpose of generating a new prototype, we are assuming you'll be using a Mac which comes with a terminal application to install, start and stop the prototype. Using a terminal is sometimes called ‘using the command line’. </p>
+
+      <h2>Installing the prototype</h2>
+        <h3>1. Run the following commands locally</h3>
+        <p>Using terminal, run the following commands below (the parts in ALL CAPS should be replaced with something specific to you):</p>
+
+        <pre><code>
+        mkdir YOUR-PROTOTYPE <br />
```

Comment:
The space before the `<br />` should be removed but it's not that big of a deal.